platform/android/export/export_plugin.cpp

```diff
@@ -2335,7 +2335,7 @@ Error EditorExportPlatformAndroid::sign_apk(const Ref<EditorExportPreset> &p_pre
 	String apksigner = get_apksigner_path();
 	print_verbose("Starting signing of the " + export_label + " binary using " + apksigner);
 	if (!FileAccess::exists(apksigner)) {
-		EditorNode::add_io_error(vformat(TTR("'apksigner' could not be found.\nPlease check the command is available in the Android SDK build-tools directory.\nThe resulting %s is unsigned."), export_label));
+		add_message(EXPORT_MESSAGE_WARNING, TTR("Code Signing"), vformat(TTR("'apksigner' could not be found. Please check the command is available in the Android SDK build-tools directory. The resulting %s is unsigned."), export_label));
```

Comment:
Reads better to me with:
```suggestion
		add_message(EXPORT_MESSAGE_WARNING, TTR("Code Signing"), vformat(TTR("'apksigner' could not be found. Please check that the command is available in the Android SDK build-tools directory. The resulting %s is unsigned."), export_label));
```